### ISSUE TITLE
PDAL packaging: file permission and UTF8 errors

### ISSUE THREAD

**Issue by neteler** [PDAL/PDAL#3090]
While trying to make PDAL an official Fedora/Centos/RHEL package (https://bugzilla.redhat.com/show_bug.cgi?id=1838686), the `rpmlint` program complains about a few issues which will be easy to fix upstream, i.e. in the main repo:

```
rpmlint PDAL-doc-2.1.0-3.fc33.noarch.rpm
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/_static/logo/Bauhaus93.ttf
PDAL-doc.noarch: W: wrong-file-end-of-line-encoding /usr/share/doc/PDAL/doc/_static/logo/sticker/front.ai
PDAL-doc.noarch: W: file-not-utf8 /usr/share/doc/PDAL/doc/_static/logo/sticker/front.ai
PDAL-doc.noarch: W: wrong-file-end-of-line-encoding /usr/share/doc/PDAL/doc/_static/logo/sticker/iheartpdal.ai
PDAL-doc.noarch: W: file-not-utf8 /usr/share/doc/PDAL/doc/_static/logo/sticker/iheartpdal.ai
PDAL-doc.noarch: W: wrong-file-end-of-line-encoding /usr/share/doc/PDAL/doc/_static/logo/sticker/sticker.ai
PDAL-doc.noarch: W: file-not-utf8 /usr/share/doc/PDAL/doc/_static/logo/sticker/sticker.ai
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-bold-webfont.eot
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-bold-webfont.svg
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-bold-webfont.ttf
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-bold-webfont.woff
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-bold-webfont.woff2
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-regular-webfont.eot
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-regular-webfont.svg
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-regular-webfont.ttf
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-regular-webfont.woff
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/miriamlibre-regular-webfont.woff2
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-bold-webfont.eot
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-bold-webfont.svg
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-bold-webfont.ttf
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-bold-webfont.woff
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-bold-webfont.woff2
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-regular-webfont.eot
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-regular-webfont.svg
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-regular-webfont.ttf
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-regular-webfont.woff
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/pdal_rtd/static/fonts/sintony-regular-webfont.woff2
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/stages/filters.statisticaloutlier.img1.png
PDAL-doc.noarch: W: spurious-executable-perm /usr/share/doc/PDAL/doc/stages/filters.statisticaloutlier.img2.png
```

I'd be happy to not workaround that in the SPEC file but to see it fixed in PDAL itself. Thanks!

**Comment by hobu** [comment]
> The library versioning adopted by PDAL is pretty unusual, i.e. in CMakeLists.txt

This was a bug that is fixed in #3042 

The permissions are changed in #3093

Your spec file should just ignore `doc/_static/logo/` entirely and not include it.

**Comment by neteler** [comment]
> > The library versioning adopted by PDAL is pretty unusual, i.e. in CMakeLists.txt
> 
> This was a bug that is fixed in #3042
> 
> The permissions are changed in #3093

Thanks for this.

> Your spec file should just ignore `doc/_static/logo/` entirely and not include it.

Right, now it will be properly built.

**Comment by neteler** [comment]
> > The library versioning adopted by PDAL is pretty unusual, i.e. in CMakeLists.txt
> 
> This was a bug that is fixed in #3042

Forgot to ask: @hobu when is the next release which includes this fix to be expected?

**Comment by hobu** [comment]
This is merged to the `2.1-maintenance` branch and will be included in 2.1.1, which is expected by the end of June.
